javatests/arcs/showcase/references/Arcs.kt

```diff
@@ -3,27 +3,68 @@
 package arcs.showcase.references
 
 import android.content.Context
-import androidx.lifecycle.Lifecycle
 import androidx.lifecycle.LifecycleOwner
 import androidx.lifecycle.LifecycleRegistry
-import arcs.android.sdk.host.AndroidHost
 import arcs.core.allocator.Allocator
-import arcs.core.common.toArcId
+import arcs.core.allocator.Arc
+import arcs.core.common.ArcId
+import arcs.core.data.Plan
+import arcs.core.host.AbstractArcHost
 import arcs.core.host.EntityHandleManager
-import arcs.core.host.SchedulerProvider
+import arcs.core.host.ParticleState
 import arcs.core.host.toRegistration
-import arcs.core.storage.StoreManager
+import arcs.core.host.SchedulerProvider
+import arcs.core.storage.ActivationFactory
 import arcs.jvm.host.ExplicitHostRegistry
 import arcs.jvm.host.JvmSchedulerProvider
 import arcs.jvm.util.JvmTime
+import arcs.sdk.Particle
 import arcs.sdk.android.storage.ServiceStoreFactory
 import arcs.sdk.android.storage.service.ConnectionFactory
 import kotlinx.coroutines.ExperimentalCoroutinesApi
 import kotlinx.coroutines.runBlocking
 import kotlinx.coroutines.sync.Mutex
 import kotlinx.coroutines.sync.withLock
+import kotlinx.coroutines.withTimeout
 import kotlin.coroutines.EmptyCoroutineContext
 
+/** Container for WriteRecipe specific things */
```

Comment:
If the intent is that this is generated by Archron or used by some Archron common lib logic down the road, perhaps worth noting in the comment.

I could be off base here, but wanted to ask while reading.